mono/mini/exceptions-amd64.c

```diff
@@ -885,7 +885,7 @@ mono_arch_exceptions_init (void)
 	}
 }
 
-#if defined(TARGET_WIN32) && !defined(DISABLE_JIT)
+#if defined(TARGET_WIN32) && !defined(DISABLE_JIT) && defined(MONO_ARCH_HAVE_UNWIND_TABLE)
```

Comment:
This should only check for MONO_ARCH_HAVE_UNWIND_TABLE as that define is already behind `#if defined(TARGET_WIN32) && !defined(DISABLE_JIT)`